### ISSUE TITLE
App does only support Portrait mode

### ISSUE THREAD

**Issue by freshNfunky** [xindawndev/RecordMyScreen-iOS10#24]
i browsed a bit the code: where exactly should i implement a landscape capturing mode in CSScreenRecorder.m

in `video_process` method or the `startRecordingScreen`

for me it's yet not clear how the `*buffer` data is comming from and how it's processed according landscape or portrait mode...

**Comment by BlazingGenerator** [comment]
Instead of doing that. Just rotate the video when you are merging audio and video together

**Comment by Taherismail17** [comment]
Did u manage to fix?

**Comment by xindawndev** [comment]
Please update to the latest version

2017-03-12 11:55 GMT+08:00 Taher Ismail <notifications@github.com>:

**Comment by Taherismail17** [comment]
yes I have updated, also the app is crashing when I press record

**Comment by BlazingGenerator** [comment]
yeah same

**Comment by xindawndev** [comment]
Please update to the latest commit

2017-03-12 16:01 GMT+08:00 BlazingGenerator <notifications@github.com>:

**Comment by xindawndev** [comment]
Please clone a new code project to test it.

2017-03-12 16:34 GMT+08:00 xindawndev <xindawndev@gmail.com>:

**Comment by xindawndev** [comment]
We updated the library,

2017-03-12 16:39 GMT+08:00 xindawndev <xindawndev@gmail.com>:
